### PR TITLE
Updated Screenshots

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates screenshots of project/tenant settings.

## Changed Pages

Lots of them 😓 

## Jira Tickets

🎫 [PEM-5236](https://spectrocloud.atlassian.net/browse/PEM-5236)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [x] No. _Please leave a short comment below about why this PR cannot be backported._

Release PR

[PEM-5236]: https://spectrocloud.atlassian.net/browse/PEM-5236?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md [hunk 1]
```diff
@@ -54,8 +54,6 @@ Use the following steps to enable OIDC SSO in Palette with Microsoft Entra ID.
 2.  Navigate to the left **Main Menu** and select **Tenant Settings**. From the **Tenant Menu**, select **SSO**, then
     **Configure**, and lastly, click on the **OIDC** tab.
 
-    ![A view of the callback URL field](/oidc-entra-id-images/user-management_saml-sso_palette-sso-with-entra-id-oidcallback.webp)
```
Comment (addetz) [inline docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md:57]: Was this removal without replacement image intended?

File: docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md [hunk 2]
```diff
@@ -105,72 +105,70 @@ exposed services in the cluster details page.
...
     message stating "OIDC configured successfully" at the top right corner.
 
-![alt_text](/keycloak/user-management_saml-sso_keycloak-09-palette-oidc-enabled.webp "Palette OIDC Enabled")
-
-17. Navigate back to the Keycloak admin console. In the left **Main Menu**, select **Users**. Fill in first name, second
+16. Navigate back to the Keycloak admin console. In the left **Main Menu**, select **Users**. Fill in first name, second
     name, and the email address of the admin user.
 
-![alt_text](/keycloak/user-management_saml-sso_keycloak-10-keycloak-admin.webp "Keycloak Admin")
+    ![alt_text](/keycloak/user-management_saml-sso_keycloak-10-keycloak-admin.webp "Keycloak Admin")
```
Comment (addetz) [inline docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md:171]: Should we write something in `alt_text` perhaps?

File: docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md [hunk 3]
```diff
@@ -197,29 +195,27 @@ Use the following steps to validate the SSO configuration.
...
-![alt_text](/keycloak/user-management_saml-sso_keycloak-14-palette-groups.webp "Palette groups")
+   ![Image of the Palette Tenant menu with Users & Teams highlighted](/keycloak/user-management_saml-sso_keycloak-14-palette-groups.webp "Palette groups")
 
 2. Provide a team name, such as `admins` but leave the **Members** field empty. Click on **Confirm** to create the team.
 
 3. Next, select the **admins** row to access the team settings page. Click on **New Project Role** and add the **Project
    Admin** role to the **Default** project. Click on **Confirm** to save the changes.
 
-![alt_text](/keycloak/user-management_saml-sso_keycloak-15-palette-project.webp "Palette Project")
+   ![Image of project roles highlighteds](/keycloak/user-management_saml-sso_keycloak-15-palette-project.webp "Palette Project")
```
Comment (addetz) [inline docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md:205]:
```suggestion
   ![Image of project roles highlighted](/keycloak/user-management_saml-sso_keycloak-15-palette-project.webp "Palette Project")
```

File: docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md [hunk 4]
```diff
@@ -197,29 +195,27 @@ Use the following steps to validate the SSO configuration.
...
    Click on the **OIDC** tab to access the OIDC configuration page. Scroll down to the **SSO Admins** section and select
    the **admins** team from the **Default Team** drop-down Menu. Click on **Save**.
 
-![alt_text](/keycloak/user-management_saml-sso_keycloak-16-palette-sso-admins.webp "Palette SSO Admins")
-
 5. Navigate back to the Keycloak admin console. From the left **Main Menu** click on **Users**. Create a user and ensure
    the field **Username**, **Email**, and **First Name** are filled out. Click on **Save** to save the user.
 
-![alt_text](/keycloak/user-management_saml-sso_keycloak-17-keycloak-user.webp "Keycloak user")
+   ![Keycloack's management portal with the User view in focus](/keycloak/user-management_saml-sso_keycloak-17-keycloak-user.webp "Keycloak user")
```
Comment (addetz) [inline docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md:214]:
```suggestion
   ![Keycloak's management portal with the User view in focus](/keycloak/user-management_saml-sso_keycloak-17-keycloak-user.webp "Keycloak user")
```

File: docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md [hunk 5]
```diff
@@ -197,29 +195,27 @@ Use the following steps to validate the SSO configuration.
...
 
 6. Next, click on the **Credentials** tab and assign the user password.
 
-![alt_text](/keycloak/user-management_saml-sso_keycloak-18-keycloak-password.webp "Keycloak password")
+   ![Keycloack's management portal with the User view in focus that lists all current users](/keycloak/user-management_saml-sso_keycloak-18-keycloak-password.webp "Keycloak password")
```
Comment (addetz) [inline docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md:218]:
```suggestion
   ![Keycloak's management portal with the User view in focus that lists all current users](/keycloak/user-management_saml-sso_keycloak-18-keycloak-password.webp "Keycloak password")
```

File: docs/docs-content/user-management/saml-sso/palette-sso-with-okta-saml.md [hunk 6]
```diff
@@ -30,33 +30,26 @@ This step is to create teams in Palette that will be used for Okta SAML, this wi
...
+    automatically from the SSO. Click **Confirm** to create the team.
 
-2. Specify `Okta Team` in the **Team name** field. You don't need to set any members now, as this will happen
-   automatically from the SSO. Click **Confirm** to create the team.
+    ![Name Palette Team](/palette-sso-with-adfs-images/how-to_palette-sso-with-adfs_name-team.webp)
 
-![Name Palette Team](/palette-sso-with-adfs-images/how-to_palette-sso-with-adfs_name-team.webp)
+3.  The list of teams displays again. Select the newly created **Okta Team** team to review its details. To give this
+    team administrative access to the entire tenant and all the projects in it, assign the **Tenant Admin** role. Select
+    **Tenant Roles** and click **+ Add Tenant Role**:
```
Comment (addetz) [inline docs/docs-content/user-management/saml-sso/palette-sso-with-okta-saml.md:46]:
```suggestion
    **Tenant Roles** and click **+ Add Tenant Role**.
```

**Review by addetz — APPROVED** [review]
Teeny things. 🚀

File: docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md (hunk 2 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md:171]:
```suggestion
    ![View of the Keycloack admin portal screen](/keycloak/user-management_saml-sso_keycloak-10-keycloak-admin.webp "Keycloak Admin")
```

Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md:171]:
```suggestion
    ![View of the Keycloack admin portal screen](/keycloak/user-management_saml-sso_keycloak-10-keycloak-admin.webp "Keycloak Admin")
```

**Pull request merged**